<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="旅程文化"
           tests="2"
           failures="1"
           errors="0"
           skipped="0"
           time="46.434">
  <properties />
            <testcase name="test2" time="18.232" classname="com.winsion.inception.test.lc.Lcwh">
                <failure type="java.lang.AssertionError"
                  message="reply expected [15022002062：rrr3] but found [0]"
              >
<![CDATA[java.lang.AssertionError: reply expected [15022002062：rrr3] but found [0]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:176)
  com.winsion.inception.test.lc.Lcwh.test2(Lcwh.java:115)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:483)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:335)
  org.testng.SuiteRunner.access$000(SuiteRunner.java:37)
  org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:369)
  org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
  java.util.concurrent.FutureTask.run(FutureTask.java:266)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  java.lang.Thread.run(Thread.java:744)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="test1" time="28.202" classname="com.winsion.inception.test.lc.Lcwh" />
      
</testsuite>
